examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>Maybe something worded like this?</p><p> </p><p>"Specifically, we'll want to mask the pixels where the network predicted <code>unlabeled</code> and avoid computing the loss for it since it doesn't contribute to to training that much" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>